.github/workflows/ci-cd.yaml

```diff
@@ -142,16 +107,23 @@ jobs:
       - name: Download Trento Web e2e artifact
```

Comment:
I think this is a leftover from your the previous asset you where creating in `setup-e2e`. As the artifact is now the binary from `build-static-binary`, a more suited name would be `Download trento binary`, wdyt?

Reply:
changed, thanks